src/luminol/correlator.py

```diff
@@ -113,4 +116,6 @@ def is_correlated(self, threshold=None):
         Compare with a threshold to determine whether two timeseries correlate to each other.
         :return: a CorrelationResult object if two time series correlate otherwise false.
         """
+        if threshold is None:
+            threshold = 0
```

Comment:
Same comment as above, why not just change the method definition?

Reply:
Yes, this is definitely better. Making the change now.